typescript/cli/src/rebalancer/strategy/MinAmountStrategy.ts

```diff
@@ -69,9 +69,6 @@ export class MinAmountStrategy extends BaseStrategy {
       (sum, chain) => sum + rawBalances[chain],
       0n,
     );
-    const minAmountType = this.config[this.chains[0]].minAmount.type;
-
-    this.validateAmounts(totalCollateral, minAmountType);
```

Comment:
@Mo-Hussain, @fzavalia 

It will no longer check for valid amounts during execution and only on startup